SimPEG/electromagnetics/natural_source/simulation.py

```diff
@@ -1,584 +1,584 @@
-import time
-import sys
-import scipy.sparse as sp
-import numpy as np
-import dask
-import dask.array as da
-import multiprocessing
-from scipy.constants import mu_0
-
-try:
-    from pymatsolver import Pardiso as SimpegSolver
-except ImportError:
-    from SimPEG import SolverLU as SimpegSolver
-
-from ...utils import mkvc, setKwargs
```

Comment:
I have installed "utils" from "https://pypi.org/project/python-utils/" After line 15 getting an error of "ValueError: attempted relative import beyond top-level package". I checked utils package it is not having any mkvc named file.
![mkvc3 2019-10-09 10-18-12](https://user-images.githubusercontent.com/31276978/66452486-466f7800-ea7e-11e9-9c92-efef17ce5f91.png)

Reply:
@nishantra, in this case, `utils` is a module of SimPEG, so the import is relative. You shouldn't need `python-utils` to run SimPEG. Does that make sense?